### PR TITLE
[cleanup] moving test sql to a yaml file for better readability

### PR DESCRIPTION
**Pull request by walterddr**
- hard-coded string needs to follow checkstyle. also it cannot be directly copied. 
- didn't use json b/c it doesn't support multiline as well as yaml

### CODE REVIEW

**Review by agavra — APPROVED** [review]
😍 yessss! can we also do that for QueryRunnerTest and make this a generic thing we can use in lots of tests 🙏

File: pinot-integration-tests/src/test/resources/ssb/ssb_query_set.yaml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    and LO_SUPPKEY = S_SUPPKEY and LO_PARTKEY = P_PARTKEY and LO_ORDERDATE = D_DATEKEY
+    and C_REGION = 'AMERICA' and S_REGION = 'AMERICA' and (D_YEAR = 1997 or D_YEAR = 1998)
+    and (P_MFGR = 'MFGR#1' or P_MFGR = 'MFGR#2') group by D_YEAR, S_NATION, P_CATEGORY
+    order by D_YEAR, S_NATION, P_CATEGORY;
+  - |
+    select D_YEAR, S_CITY, P_BRAND1, sum(LO_REVENUE - LO_SUPPLYCOST) as profit
+    from lineorder, dates, customer, supplier, part where LO_CUSTKEY = C_CUSTKEY
+    and LO_SUPPKEY = S_SUPPKEY and LO_PARTKEY = P_PARTKEY and LO_ORDERDATE = D_DATEKEY
+    and C_REGION = 'AMERICA' and S_NATION = 'UNITED STATES' and (D_YEAR = 1997 or D_YEAR = 1998)
+    and P_CATEGORY = 'MFGR#14' group by D_YEAR, S_CITY, P_BRAND1 order by D_YEAR, S_CITY, P_BRAND1 ;
```
Comment (Jackie-Jiang) [inline pinot-integration-tests/src/test/resources/ssb/ssb_query_set.yaml:94]: (nit) Empty line in the end

**Review by Jackie-Jiang — APPROVED** [review]

File: pinot-integration-tests/src/test/resources/ssb/ssb_query_set.yaml (hunk 1 above)
Comment (walterddr) [reply pinot-integration-tests/src/test/resources/ssb/ssb_query_set.yaml:94]:
```suggestion
    and P_CATEGORY = 'MFGR#14' group by D_YEAR, S_CITY, P_BRAND1 order by D_YEAR, S_CITY, P_BRAND1 ;

```

**Comment by walterddr** [comment]
> 😍 yessss! can we also do that for QueryRunnerTest and make this a generic thing we can use in lots of tests 🙏

good idea. although there's more to improve in QueryRunnerTest for example
- better data randomness, currently they are hard-coded
- better data type coverage, current only 2 numeric, 2 string and 1 timestamp
- better table schema, currently all are the same.
let's create an issue to track this?

**Pull request merged**